components/eam/src/physics/cam/physpkg.F90

```diff
@@ -2785,6 +2790,7 @@ subroutine phys_timestep_init(phys_state, cam_out, pbuf2d)
   call prescribed_ozone_adv(phys_state, pbuf2d)
   call prescribed_ghg_adv(phys_state, pbuf2d)
   call prescribed_aero_adv(phys_state, pbuf2d)
+  call read_spa_data_adv(phys_state, pbuf2d)
```

Comment:
call only if SPA is active

Reply:
I agree, but I'm surprised that the other function calls here aren't within if statements. Maybe the typical paradigm is to always call but to return without doing anything if that condition isn't met?